docs/product-highlights-and-benefits/connecting-to-a-remote-pieces-os-instance.mdx

```diff
@@ -11,6 +11,8 @@ This guide will walk you through how to connect to a remote Pieces OS instance u
 
 > - This is currently only available for the Pieces for Developers VS Code extension. Functionality for the rest of the Pieces for Developers ecosystem is coming soon.
 
+<Info>If Pieces OS is running on the same machine as VS Code, there should be no requirement to use this feature. Please verify these steps are required for your set up before following this guide. This custom URL is not the same thing as your Pieces Cloud custom url.</Info>
```

Comment:
I would probably put this right under the "Connecting to a Remote Pieces OS Instance" header so its one of the first things people read. Thoughts?

Reply:
I have a habit of skipping at least 1/3 of the webpage cause most of the time it's just garbage info up at the top but I'm down